src/Components/Form/SelectMenuV2.tsx

```diff
@@ -68,7 +68,7 @@ const SelectMenuV2 = <T, V>(props: SelectMenuProps<T, V>) => {
       >
         {({ open }) => (
           <>
-            <Listbox.Label className="sr-only">
+            <Listbox.Label className="sr-only !relative">
```

Comment:
And, I don't think so this modification helps. Am I missing something?

Reply:
`sr-only` gives absolute positioning for the label, which kinda causes this issue, to be more specific it is because of the select menus in the prescription and the medication builder.